tests/performance/log-results.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const https = require( 'https' );
+const [ token, branch, hash, timestamp ] = process.argv.slice( 2 );
+
+const resultsFiles = [
+	{
+		file: 'home-block-theme.test.results.json',
+	},
+	{
+		file: 'home-classic-theme.test.results.json',
+	},
+];
+
+const performanceResults = resultsFiles.map( ( { file } ) =>
+	JSON.parse(
+		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+	)
+);
+
+function median( array ) {
```

Comment:
Rather than duplicating this functionality here and also in the `results.js` file let's consolidate it to a single `utils.js` file that can be required in these two files.

Reply:
Good catch. Updated